apps/frontend/src/pages/auth/components/update-password-form/update-password-form.tsx

```diff
@@ -0,0 +1,100 @@
+import { Button, Input } from "~/libs/components/components.js";
+import { AppTitle, DataStatus } from "~/libs/enums/enums.js";
+import {
+	useAppForm,
+	useAppSelector,
+	useAppTitle,
+	useCallback,
+	useState,
+} from "~/libs/hooks/hooks.js";
+import { type AuthUpdatePasswordRequestDto } from "~/modules/auth/auth.js";
+import { authUpdatePasswordValidationSchema } from "~/modules/auth/auth.js";
+
+import { DEFAULT_UPDATE_PASSWORD_IN_PAYLOAD } from "./libs/constants.js";
+import styles from "./styles.module.css";
+
+type Properties = {
+	onSubmit: (payload: AuthUpdatePasswordRequestDto) => void;
+	token: string;
+};
+
+const UpdatePasswordForm: React.FC<Properties> = ({
+	onSubmit,
+	token,
+}: Properties) => {
+	const { control, errors, handleSubmit } =
+		useAppForm<AuthUpdatePasswordRequestDto>({
+			defaultValues: {
+				...DEFAULT_UPDATE_PASSWORD_IN_PAYLOAD,
+				token,
+			},
+			validationSchema: authUpdatePasswordValidationSchema,
+		});
+
+	const authDataStatus = useAppSelector(({ auth }) => {
+		return auth.dataStatus;
+	});
+
+	const [isPasswordVisible, setIsPasswordVisible] = useState<boolean>(false);
+
+	const handleFormSubmit = useCallback(
+		(event_: React.BaseSyntheticEvent): void => {
+			void handleSubmit(onSubmit)(event_);
+		},
+		[handleSubmit, onSubmit],
+	);
+
+	const handleChangePasswordVisibility = useCallback(() => {
+		setIsPasswordVisible(!isPasswordVisible);
+	}, [isPasswordVisible]);
+
+	useAppTitle(AppTitle.UPDATE_PASSWORD);
+
+	const isLoading = authDataStatus === DataStatus.PENDING;
+
+	return (
+		<form className={styles["form"]} onSubmit={handleFormSubmit}>
+			<div>
+				<h2 className={styles["title"]}>Update password</h2>
+			</div>
+			<Input
+				className="visually-hidden"
+				control={control}
+				errors={errors}
+				hasVisuallyHiddenLabel
+				label="Token"
+				name="token"
+				type="text"
```

Comment:
```suggestion
				type="hidden"
```
you need to use hidden for such cases. and it can be just a `input` html tag

Reply:
@what1s1ove what about `control`?

Reply:
I have added `hidden` type to `Input` component